<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 18pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Personal Accounting System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Segoe Script&quot;;</string>
         </property>
         <property name="text">
          <string>Balance</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Bahnschrift Condensed</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="text">
          <string>123</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="comboBox_period">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Total</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Year</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Month</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Week</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_currency">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Yuan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Euro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>US Dollar</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBox_category">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Work</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>School</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Refund</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entertainment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Others</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transportation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Projects</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWebEngineView" name="webEngineView">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>250</height>
        </size>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
